lib/version_utils.pm

```diff
@@ -531,7 +533,7 @@ configuration, otherwise returns false (0).
 
 =cut
 sub has_license_on_welcome_screen {
-    return 1 if is_caasp('caasp');
+    return 1 if is_caasp('caasp') || is_caasp('suse_microos');
```

Comment:
What about `is_caasp` without arguments?

Reply:
There are 3 cases: `is_caasp('caasp')`, `is_caasp('microos')` and now `is_caasp('suse_microos')`. 
If I change this line here to `is_caasp` it will hit the 3 cases, and I think opensuse MicroOS doesn't have the license on welcome screen, so it should not return 1. 

We for sure need to make this function clearer, as the next task maybe.

Reply:
Although both have licences on the welcome screen, oS MicroOS doesn't show accept licence checkbox.
It applies for Leap and TW based MicroOS. IMHO `is_caasp && !is_opensuse` would describe the situation more precisely. 
I haven't found any `DISTRI=caasp` in o3 defined, `is_caasp('caasp')` seems like completely abandoned.

Reply:
Interestingly `is_opensuse` would return 1, cause `DISTRI=microos` ... not sure if it would be better to create a new product (DISTRI=suse_microos) for this.. but I'd like to avoid creating too many new stuff like a new symlink directory, etc

Reply:
Ah, I haven't expected that. Thanks for checking the code. I am fine with your current proposal then and later on we can rethink it.